addons/point_of_sale/static/src/js/models.js

```diff
@@ -2207,9 +2208,10 @@ exports.Orderline = Backbone.Model.extend({
             var product_taxes = [];
 
             _(taxes_ids).each(function(el){
-                product_taxes.push(_.detect(taxes, function(t){
+                var tax = _.detect(taxes, function(t){
                     return t.id === el;
-                }));
+                });
+                product_taxes.push.apply(product_taxes, self._map_tax_fiscal_position(tax, self.order));
             });
```

Comment:
I agree with this fix. But I would like to suggest to extract a method out of this highlighted code (or a portion of it) because I saw a complete copy.

Reply:
Thanks for the review, I've extracted the tax mapping into the method `get_taxes_after_fp`
Hope this is what you had in mind